oneflow/python/ops/constant_op.py

```diff
@@ -120,4 +266,41 @@ def zeros_like(
     dtype: Optional[dtype_util.dtype] = None,
     name: Optional[str] = None,
 ) -> remote_blob_util.BlobDef:
+    """This operator creates a Blob with all elements set to `0` that has the same shape as `like`.
```

Comment:
This operator creates a Blob that has the same shape as `like` whose all elements are set to `0`